src/shopilingus/Main.java

```diff
@@ -0,0 +1,18 @@
+package shopilingus;
+
+public class Main {
+    public static void main(String[] args) {
+
+        ShoppingCenter galeriaRzeszów = new ShoppingCenter("Galeria Rzeszów", "ul.Józefa Piłsudskiego");
```

Comment:
we can use polish letters in strings, but we should avoid them in variable names 

`galeriaRzeszów` -> `galeriaRzeszow`